src/Logic.hs

```diff
@@ -368,7 +368,10 @@ tryIntegratePullRequest pr state =
       Nothing  -> do
         -- If integrating failed, perform no further actions but do set the
         -- state to conflicted.
-        leaveComment pr "Failed to rebase, please rebase manually."
+        leaveComment pr $ Text.unlines
+          [ "Failed to rebase, please rebase manually using"
+          , "> `git rebase --interactive --autosquash ...`"
```

Comment:
We could even include the full command, as [we know the branch name](https://github.com/channable/hoff/blob/a4531e77843f6dfcb76edde60e6a57131ec9f7c9/src/Project.hs#L90). The actual command [is this one](https://github.com/channable/hoff/blob/a4531e77843f6dfcb76edde60e6a57131ec9f7c9/src/Git.hs#L251-L254), but it rebases the specific sha, not the branch name. So the user-friendly version would be

    git rebase --interactive --autosquash origin/{targetBranch} {prBranch}

The target branch is in [the project config](https://github.com/channable/hoff/blob/a4531e77843f6dfcb76edde60e6a57131ec9f7c9/src/Configuration.hs#L31), which is not yet available here. But maybe that was a premature generalization, and we can just hard-code it to be `master`.

Reply:
This should be fixed now, the required information is returned in the error case.